variants/MALYANM200_F103CB/ldscript.ld

```diff
@@ -0,0 +1,167 @@
+/*
+*****************************************************************************
+**
+
+**  File        : LinkerScript.ld
+**
+**  Abstract    : Linker script for Malyan M200 Device with
+**                120Byte FLASH, 20KByte RAM
+**
+**                Set heap size, stack size and stack location according
+**                to application requirements.
+**
+**                Set memory bank area and size if external memory is used.
+**
+**  Target      : STMicroelectronics STM32
+**
+**
+**  Distribution: The file is distributed as is, without any warranty
+**                of any kind.
+**
+**  (c)Copyright Ac6.

```

Comment:
Please update the license as done here:
https://github.com/stm32duino/Arduino_Core_STM32/commit/e7af82b63ecf7affbe5dd19eacceb43bb43019fc

Reply:
Fixed, thanks. On a side note - the 070 LD script has no header of any sort. What should it be?

Reply:
The same if you get it thanks Cube MX. Else no worry about that.
Main things is this License should not appears as too restrictive. We have correct this internally so ok to change it.